src/modules/observation/EditObservationContainer.js

```diff
@@ -25,7 +25,9 @@ const mapDispatchToProps = (dispatch) => ({
       dispatch(addControl(id, controlState, observationState, museumId, {
         onSuccess: () => {
           hashHistory.goBack();
-          emitSuccess( { type: 'saveSuccess', message: I18n.t('musit.observation.page.messages.saveSuccess') });
+          emitSuccess( { type: 'saveSuccess', message: controlState ?
```

Comment:
du behøver ikke en if sjekk her. Vi vil at det alltid skal vises en melding for kontroll antar jeg. Edit observation er per def en kontroll. Ikke en observasjon. Så du kan ta vekk  I18n.t('musit.observation.page.messages.saveSuccess') og bare si emitSuccess( { type: 'saveSuccess', message: I18n.t('musit.newControl.saveControlSuccess') });